docs/vm/backup-restore.md

```diff
@@ -68,3 +68,42 @@ You can choose to either delete or retain the previous volumes. By default, all
 1. Click `Create`.
 
 The restore process can be viewed from the `Virtual Machines` page.
+
+## Restore a new VM on another Harvester cluster
+
+_Available as of v1.0.0_
+
+You can migrate a backup to another Harvester cluster and create a new VM based on it.
+
+**Requirements:** The VM image name and content must be the same as configured in the cluster where the backup was migrated from.
+
+### Upload the same VM images to a new cluster
+
+1. Get the VM image names in the current cluster.
+```
+> kubectl get vmimage -o json | jq '.items[].metadata.name'
```

Comment:
```
$ kubectl get vmimages -A
NAMESPACE   NAME                               DISPLAY-NAME                              SIZE         AGE
default     image-79hdq                        focal-server-cloudimg-amd64.img           566886400    5h36m
default     image-l7924                        harvester-v1.0.0-rc2-amd64.iso            3964551168   137m
default     image-lvqxn                        opensuse-leap-15.3.x86_64-nocloud.qcow2   568524800    5h35m
```

Check the existing image name (normally starts with `image-`) and create the same one on the new cluster.

Reply:
Updated it. Thank you.